core/pom.xml

```diff
@@ -167,6 +167,31 @@
 					</dependency>
 				</dependencies>
 			</plugin>
+			<plugin>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<extensions>true</extensions>
+				<executions>
+					<execution>
+						<id>jar</id>
+						<configuration>
+							<bnd><![CDATA[
+				                Bundle-Copyright: Copyright (c) (2019) Contributors to the Eclipse Foundation.
+				                Git-Descriptor: ${system-allow-fail;git describe --dirty --always --abbrev=9}
+				                Git-SHA: ${system-allow-fail;git rev-list -1 --no-abbrev-commit HEAD}
+				                SPDX-License-Identifier: ${project.licenses[0].name}
```

Comment:
Isn't it sufficient to have the `Bundle-License` that BND-tools generates by default?

Reply:
Same as above. If there is no real benefit (i.e. a relevant tool that can just use this), then I'm in favor of dropping this too.

Reply:
removed